<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPlayer</class>
 <widget class="QWidget" name="NewPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="enterButon">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>410</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>511</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="firstName_display">
        <property name="text">
         <string>First Name : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="firstName_input"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="firstName_display_2">
        <property name="text">
         <string>Last Name : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lastName_input"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="firstName_display_3">
        <property name="text">
         <string>Address : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="address_input"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
